### ISSUE TITLE
Conv3d

### ISSUE THREAD

**Issue by shlomi-amitai** [tensorflow/tensorflow#42863]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
- TensorFlow installed from (source or binary):
- TensorFlow version (or github SHA if from source):

**Provide the text output from tflite_convert**

Some of the operators in the model are not supported by the standard TensorFlow Lite runtime. If those are native TensorFlow operators, you might be able to use the extended runtime by passing --enable_select_tf_ops, or by setting target_ops=TFLITE_BUILTINS,SELECT_TF_OPS when calling tf.lite.TFLiteConverter(). Otherwise, if you have a custom implementation for them you can disable this error with --allow_custom_ops, or by setting allow_custom_ops=True when calling tf.lite.TFLiteConverter(). Here is a list of builtin operators you are using: CONCATENATION, CONV_2D, PACK, SPLIT, SUM. Here is a list of operators for which you will need custom implementations: Conv3D.  

**Standalone code to reproduce the issue** 
Provide a reproducible test case that is the bare minimum necessary to generate
the problem. If possible, please share a link to Colab/Jupyter/any notebook.

tensorflow.keras.layers.Conv3D(3, 3, padding='same')(x)

Also, please include a link to a GraphDef or the model if possible.

**Any other info / logs**

Include any logs or source code that would be helpful to diagnose the problem.
If including tracebacks, please include the full traceback. Large logs and files
should be attached.

**Comment by Saduf2019** [comment]
@shlomi-amitai 
Please provide with simple stand alone code or a colab gist with the error faced for us to analyse, the tf version on which the error is faced.

**Comment by ymodak** [comment]
`Conv3d` is WIP and not supported for TF Lite yet. See tracker https://github.com/tensorflow/tensorflow/issues/21526 to know more.

**Comment by abattery** [comment]
As @ymodak said, TFLite builtin operator set does not cover the Conv3D case. However, you can convert Conv3D op with the Select TF ops approach:

https://www.tensorflow.org/lite/guide/ops_select
https://www.tensorflow.org/lite/guide/reduce_binary_size

**Comment by shlomi-amitai** [comment]
Thanks, 
is converting Conv3d with TF ops possible also in tf 1.14 /1.15? if so, could you send me a pointer?

**Comment by abattery** [comment]
You can use the above Select TF operator option in TF 1.15 version as well. Or you can export your model to a saved model directory under TF 1.15 version and you can try the TFLite conversion with the recent TF version since TFLite converter can support TF 1.x's saved model directory format.

For the TF 1.x converter API, you can refer to the following document: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/g3doc/r1/convert/index.md

**Comment by Saduf2019** [comment]
@shlomi-amitai 
Please feel free to close this issue if resoved.

**Comment by shlomi-amitai** [comment]
Sure, just another last thing, I tried to convert with TF v1.14 
set "converter.target_ops = [tf.lite.OpsSet.TFLITE_BUILTINS, tf.lite.OpsSet.SELECT_TF_OPS"

but failed with 
"F tensorflow/lite/toco/graph_transformations/resolve_constant_transpose.cc:142] Check failed: op->perm.size() <= 4 (5 vs. 4)"

while "Converting unsupported operation: Conv3D"

Are you familiar with this?

**Comment by shlomi-amitai** [comment]
thanks!
